consensus/src/experimental/buffer_manager.rs

```diff
@@ -190,6 +191,7 @@ impl BufferManager {
             self.buffer.len() + 1,
         );
         counters::NUM_BLOCKS_IN_PIPELINE.add(ordered_blocks.len() as i64);
+        counters::NUM_ORDERED_BLOCKS_IN_PIPELINE.add(ordered_blocks.len() as i64);
```

Comment:
hmm, this feels redundant to the one above, if we want to check whether it's executed or ordered, we should just add `status` to the same metrics

Reply:
Hmm..adding status on the metrics will be confusing as we want to track both number of blocks in the pipeline and number of ordered blocks in the pipeline. They are not mutually exclusive - so adding a status dimension won't work? Am I missing something?